test/topology/test_mutation_schema_change.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Reproducer for a failure during lwt operation due to missing of a column mapping in schema history table.
+"""
+import asyncio
+import logging
+import time
+from test.pylib.rest_client import inject_error_one_shot, inject_error
+from test.pylib.util import wait_for
+import pytest
+from cassandra.cluster import Cluster, ConsistencyLevel  # type: ignore # pylint: disable=no-name-in-module
+from cassandra.query import SimpleStatement              # type: ignore # pylint: disable=no-name-in-module
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_mutation_schema_change(manager, random_tables):
+    """
+        Cluster A, B, C
+        create table
+        stop C
+        change schema + do lwt write + change schema
+        stop B
+        start C
+        do lwt write to the same key through C
+    """
+    server_a, server_b, server_c = await manager.running_servers()
+    t = await random_tables.add_table(ncolumns=5, pks=1)
+    manager.driver_close()
+    # Reduce the snapshot thresholds
+    await manager.mark_dirty()
+    errs = [inject_error_one_shot(manager.api, s.ip_addr, 'raft_server_snapshot_reduce_threshold')
+            for s in [server_a, server_b, server_c]]
+    await asyncio.gather(*errs)
+
+
+    logger.info("Stopping C %s", server_c)
+    await manager.server_stop_gracefully(server_c.server_id)
+    await manager.driver_connect()
+
+    async with inject_error(manager.api, server_b.ip_addr, 'paxos_error_before_learn'):
+        await t.add_column()
+        ROWS = 1
+        seeds = [t.next_seq() for _ in range(ROWS)]
+        stmt = f"INSERT INTO {t} ({','.join(c.name for c in t.columns)}) " \
+               f"VALUES ({', '.join(['%s'] * len(t.columns))}) "           \
+               f"IF NOT EXISTS"
+        query = SimpleStatement(stmt, consistency_level=ConsistencyLevel.ONE)
+        for seed in seeds:
+            logger.info("INSERT row seed %s", seed)
+            await manager.cql.run_async(query, parameters=[c.val(seed) for c in t.columns])
+        await t.add_column()
+
+    manager.driver_close()
+
+    logger.info("Stopping B %s", server_b)
+    await manager.server_stop_gracefully(server_b.server_id)
+    logger.info("Starting C %s", server_c)
+    await manager.server_start(server_c.server_id)
+
+    async def server_c_sees_another_server():
+        alive_nodes = await manager.api.get_alive_endpoints(server_c.ip_addr)
+        if len(alive_nodes) > 1:
+            return True
+    await wait_for(server_c_sees_another_server, time.time() + 45, period=.1)
```

Comment:
This wait ensures that CQL requests don't fail, right? It's so generally useful, it should be an option to server_start().